### ISSUE TITLE
Strange JUnit vintage test name rendering

### ISSUE THREAD

**Issue by ennru** [maichler/sbt-jupiter-interface#33]
Now that I figured out to make Jupiter Vintage Engine to run my JUnit 4 tests, I wonder if the test name rendering could be improved.

I get
```
[info] Test run started (JUnit Vintage)
[info] Test docs.javadsl.AtLeastOnceTestconsumeOneProduceMany(docs.javadsl.AtLeastOnceTest) started
[info] Test docs.javadsl.AtLeastOnceTestconsumerOneProduceConditional(docs.javadsl.AtLeastOnceTest) started
```
where the test methods are `consumeOneProduceMany` and `consumerOneProduceConditional`.

**Comment by maichler** [comment]
Wow, that sure looks awful !

Could you please check test name rendering with a locally published build of #34 (Branch @wip-ticket-33).  

The output formatting had to be revised quite a lot. Took a bit of time, sorry for that!

**Comment by ennru** [comment]
👍 I tried it on my project and the Vintage test names now look just like the Jupiter test names. 
Your changes seem to be quite big. While at it, would you fix #21 as well?

**Comment by maichler** [comment]
Great, thanks !

> Your changes seem to be quite big.

Mostly tests ensuring that I do not break anything ;)

> While at it, would you fix #21 as well?

Please see #21.
